src/me/Pseudo/DecisionGraph/Main.java

```diff
@@ -1,80 +1,89 @@
 package me.Pseudo.DecisionGraph;
 
 import java.io.File;
-import java.util.ArrayList;
 import java.util.Scanner;
 
 public class Main {
 
 	public static void main(String[] args) {
-		// Execution arguments from cmd line
-		if(args.length == 2) {
+		
+		// traverse <file>
+		if(args.length == 2 || args.length == 3) {
 			
-			if(args[0].equalsIgnoreCase("traverse") || args[0].equalsIgnoreCase("tv")) {
-				
-				DecisionGraph dg = null;
-				try {
-					dg = new DecisionGraph(new File(args[1]));
-				}catch(Exception e) {
-					e.printStackTrace();
-				}
```

Comment:
Old version did leg-work in the constructor and returned a ridiculous amount of Exceptions (which have been deleted). I've since made the constructor `private` and instead users should make use of the static method:
```java
DecisionGraph.fromFile(File f);
```